gapis/api/vulkan/api/renderpass_framebuffer.api

```diff
@@ -121,39 +121,118 @@ cmd void vkDestroyFramebuffer(
 // Renderpass //
 ////////////////
 
-@internal class SubpassDescription {
-  @unused VkSubpassDescriptionFlags        Flags
-  @unused VkPipelineBindPoint              PipelineBindPoint
-  @unused map!(u32, VkAttachmentReference) InputAttachments
-  @unused map!(u32, VkAttachmentReference) ColorAttachments
-  @unused map!(u32, VkAttachmentReference) ResolveAttachments
-  @unused ref!VkAttachmentReference        DepthStencilAttachment
-  @unused map!(u32, u32)                   PreserveAttachments
+@internal enum RenderPassVersion : u32 {
```

Comment:
I would call it `kind`, `type`, or `flavor` rather than version.

Reply:
Well it's due to the version of the function. I don't know if they have a specific name for it in the spec but `kind` or `type` is actually misleading because it does **not** create a different kind of renderpass, it's just another function to create the same `kind` or `type` of renderpass. And I don't know what does flavor even mean in this context.

I think version is quite fine.

Reply:
@pmuetschard

Reply:
I would say they are different types, which is why we have this field, but I'm OK to keep `version` if you feel strongly about it. I was suggesting something different, because to me version made it sound like you could create "another version of this renderpass" and not so much "you used this version of the `vkCreateRenderPass` function to create it".

Reply:
I agree that what you say is correct but I think none of the options here makes it any better. Because for example, using "type" may imply that if you create a `RenderPass` with `VkCreateRenderPass2` you cannot use `VkBeginRenderPass`  or vice versa but it's not true. This is a completely valid use case:

`VkCreateRenderPass2`
`VkBeginRenderPass`
`VkNextSubpass2`
`VkEndRenderPass2KHR`

So, the RenderPass we create or use are the exact same RenderPass in all of these functions, it's just the API difference. 

But I agree that it is hard to state this clearly.